plugins/uefi/uefi.conf

```diff
@@ -1,5 +1,9 @@
 [uefi]
 
+# the shim loader is required to chainload the fwupd EFI binary unless
+# the fwupd.efi file has been self-signed manually
+RequireShim=true
```

Comment:
I think you should rename this to "RequireShimForSecureBoot".  Reasoning being that shim won't be used in the non-secure boot case at all.

Reply:
Sure, fixed.